edited/6m_caltrans_activities.py

```diff
@@ -77,7 +77,7 @@ def CalTrans(CalTrans_act_ln_standardized_20220712b="C:\\Users\\sageg\\Documents
     Updated_Input_Table_10_ = arcpy.management.CalculateField(in_table=Updated_Input_Table_6_, field="PRIMARY_FUNDING_ORG", expression="\"GENERAL_FUND\"", expression_type="PYTHON3", code_block="", field_type="TEXT", enforce_domains="NO_ENFORCE_DOMAINS")[0]
 
     # Process: Calculate Treatment ID (Calculate Field) (management)
-    Updated_Input_Table_45_ = arcpy.management.CalculateField(in_table=Updated_Input_Table_10_, field="TRMTID_USER", expression="!IMMS_Unit_ID!", expression_type="PYTHON3", code_block="", field_type="TEXT", enforce_domains="NO_ENFORCE_DOMAINS")[0]
+    Updated_Input_Table_45_ = arcpy.management.CalculateField(in_table=Updated_Input_Table_10_, field="TRMTID_USER", expression="!IMMS_ID!+'-'+!COUNTY![:5]+'-'+!REGION![:3]+'-'+!IN_WUI![:3]", expression_type="PYTHON3", code_block="", field_type="TEXT", enforce_domains="NO_ENFORCE_DOMAINS")[0]
```

Comment:
Instead of 'IMMS_ID', needs to be 'IMMS_Unit_ID' (like original) - this line is run on the point data which has that field as 'IMMS_Unit_ID', polyline data is 'IMMS_ID'.

Subscripts throw error: type 'nonetype' not subscriptable. 
Remove subscripts then get error: can only concatenate str (not "NoneType") to str
Remove (+'-'+!COUNTY![:5]+'-'+!REGION![:3]+'-'+!IN_WUI![:3])  - runs without error